Packs/Core/Playbooks/playbook-A_source_host_created_uncommon_remote_scheduled_task_README.md

```diff
@@ -0,0 +1,65 @@
+This playbook handles "Uncommon remote scheduled task creation" alerts, which are generated on the source host that created the remote scheduled task.
```

Comment:
need to fix first sentence as it handles single alert alert not alerts and 
is not are

Reply:
Done